manifests/v/vim/vim/9.1.1144/vim.vim.installer.yaml

```diff
@@ -15,6 +15,7 @@ UpgradeBehavior: install
 ProductCode: Vim 9.1
 ReleaseDate: 2025-02-23
 ElevationRequirement: elevatesSelf
+Commands: vim

```

Comment:
```suggestion
Commands: 
  vim

```

Reply:
I did this in the web UI, so I'm not sure if I got the spacing correct. Example manifest: https://github.com/microsoft/winget-pkgs/blob/9544665b93c27b4cfe1559082df378c0213e30d3/manifests/k/k3d/k3d/5.8.3/k3d.k3d.installer.yaml#L7